lib/wadler/print.rb

```diff
@@ -11,6 +11,7 @@ class Wadler
     attr_reader :out
     attr_reader :tokens
     attr_reader :width
+    attr_reader :whitespace
```

Comment:
If you care about the alphabetical order (which seems to be the case), it's not sorted here and below.